httplib.h

```diff
@@ -2121,7 +2125,7 @@ inline std::string file_extension(const std::string &path) {
   std::smatch m;
   static auto re = std::regex("\\.([a-zA-Z0-9]+)$");
   if (std::regex_search(path, m, re)) { return m[1].str(); }
-  return std::string();
+  return {};
```

Comment:
What clang-tidy diagnostic did you address here exactly (just curious, never saw this before)